Splash/.gitignore

```diff
@@ -7,23 +7,7 @@ xcuserdata/
 
 ## Debugger
 xcdebugger/
-
-## compatibility with Xcode 8 and earlier (ignoring not required starting Xcode 9)
-*.xcscmblueprint
-*.xccheckout
-
-## compatibility with Xcode 3 and earlier (ignoring not required starting Xcode 4)
-build/
-DerivedData/
-*.moved-aside
-*.pbxuser
-!default.pbxuser
-*.mode1v3
-!default.mode1v3
-*.mode2v3
-!default.mode2v3
-*.perspectivev3
-!default.perspectivev3
+Breakpoints_v2.xcbkptlist
```

Comment:
Not sure if this will work as intended. 
Giving another reference: https://medium.com/codex/create-git-ignore-file-for-swift-and-xcode-development-d7faaf58893c